packages/desktop-client/src/components/accounts/Reconcile.jsx

```diff
@@ -102,16 +102,16 @@ export function ReconcileMenu({ account, onReconcile, onClose }) {
     query: balanceQuery.query.filter({ cleared: true }),
   });
   const format = useFormat();
+  const [inputValue, setInputValue] = useState(null);
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏clicking "reconcile" button with the initial value (without changing the numbers) does nothing. I think we're missing an initial value in this `useState`

Reply:
Nice catch. Should now be fixed :)